### DISCUSSION TITLE
is there any way to stringify objects in json.hpp

### DISCUSSION THREAD

**Discussion by Mouradouchane** [nlohmann/json#2345] [Q&A] [answered]
Hi guys i use this library for first time and work it perfectly with parsing json files 
but when i want to stringify some objects and then add him to json file 
i never get any way to do that if someone have any idea or example please share it with me

**Comment by nlohmann** [comment, marked as answer]
I need to understand better what you try to achieve. https://github.com/nlohmann/json#serialization--deserialization gives a good overview of how to parse (deserialize) and stringify (serialize).

**Reply by Mouradouchane** [reply]
i found solution for my problem , thanks for your answer @nlohmann  .
